### ISSUE TITLE
master and next diverging?

### ISSUE THREAD

**Issue by DarwinAwardWinner** [dimitri/el-get#1408]
@yyr created the next branch in response to #1368. However, now I see merges on both master and next, so they are diverging. From the logs, it looks like @yyr's manual merges are happening in next and the pull request "pressing the merge button" merges are going into master. So, is this intended? From the logs it looks like it might be code changes merging into next and recipe updates merging into master, which I think is reasonable. I just want to make sure that it's intentional and not by accident. (Just close this issue if you know what you're doing.)

Basically, now that we have both master and next, we should decide what the purpose of each is and state it explicitly so that other devs know what to put on which branch.
